angular-workspace/projects/ni/spright-angular/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+
+module.exports = {
+    extends: '../../../.eslintrc.js',
+    ignorePatterns: [
+        '!**/*',
+        '**/dist'
+    ],
+    plugins: ['jsdoc'],
+    overrides: [
+        {
+            files: [
+                '*.ts'
+            ],
+            parserOptions: {
+                project: [
+                    './tsconfig.lib.json',
+                    './tsconfig.spec.json'
+                ],
+                tsconfigRootDir: __dirname,
+                createDefaultProgram: true
+            },
+            rules: {
```

Comment:
Any reason not to share our Angular ESLint config between spright and nimble? Could either:
1.  move this into the angular workspace config but change the matching pattern to only target the two libraries
2. move this to the private eslint config package

Reply:
I went with a variant of 1. Now the `*-angular` package configs are nearly empty, and the example app does a little bit of overriding.